networkx/readwrite/nx_yaml.py

```diff
@@ -29,9 +29,9 @@ def write_yaml(G_to_be_yaml, path_for_yaml_output, **kwds):
 
     Parameters
     ----------
-    G : graph
+    G_to_be_yaml : graph
```

Comment:
+1 for fixing, though these functions are deprecated and will be removed soon. Nice catch nonetheless!